src/DependencyInjection/Configuration.php

```diff
@@ -99,6 +99,23 @@ public function getConfigTreeBuilder()
                         ->booleanNode('disable_ip_check')
                             ->defaultFalse()
                         ->end()
+                        ->arrayNode('autologin')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->scalarNode('cookie_name')
+                                    ->cannotBeEmpty()
+                                    ->defaultValue('FE_AUTO_LOGIN_MULTI')
+                                ->end()
+                                ->integerNode('cookie_lifetime')
+                                    ->min(0)
+                                    ->defaultValue(7776000)
+                                ->end()
+                                ->scalarNode('formfield_name')
```

Comment:
Probably just best practice because Symfony allows to configure field names of the form login as well.